test/extended/util/client.go

```diff
@@ -67,25 +71,28 @@ import (
 // CLI provides function to call the OpenShift CLI and Kubernetes and OpenShift
 // clients.
 type CLI struct {
-	execPath           string
-	verb               string
-	configPath         string
-	guestConfigPath    string
-	adminConfigPath    string
-	username           string
-	globalArgs         []string
-	commandArgs        []string
-	finalArgs          []string
-	namespacesToDelete []string
-	stdin              *bytes.Buffer
-	stdout             io.Writer
-	stderr             io.Writer
-	verbose            bool
-	showInfo           bool
-	withoutNamespace   bool
-	withoutKubeconf    bool
-	asGuestKubeconf    bool
-	kubeFramework      *e2e.Framework
+	execPath               string
+	verb                   string
+	configPath             string
+	currentWs              workSpace
+	guestConfigPath        string
+	adminConfigPath        string
+	homeSeverURL           string
```

Comment:
typo in homeServerURL or is this by design ?

Reply:
`URL` should be a special noun like JSON, just obey the golint check rules.

Reply:
Use orgServerURL instead as generally uses will create ws under org ws.